tests/adapters/module-service.test.ts

```diff
@@ -3,10 +3,26 @@ import { getTestRoute } from "../fixtures/get-test-route.js"
 import { createModuleService } from "src/adapters/module-service.js"
 import { createEdgeSpecFromRouteMap } from "src/serve/create-node-server-from-route-map.js"
 
+const defaultSpecs = {
+  globalSpec: {
+    apiName: "hello-world",
+    productionServerUrl: "https://example.com",
+
+    authMiddlewareMap: {},
+    globalMiddlewares: [],
+  },
+  routeSpec: {
+    auth: "none",
+    methods: ["GET", "POST"],
+  },
+} as const
```

Comment:
`const` needed when passing spec as a variable rather than inline?

Reply:
Need to check this 🤔

Reply:
yeah unfortunately 😢